op-node/rollup/derive/channel_out_test.go

```diff
@@ -52,7 +52,7 @@ var channelTypes = []struct {
 	{
 		Name: "Span",
 		ChannelOut: func(t *testing.T) ChannelOut {
-			cout, err := NewSpanChannelOut(0, big.NewInt(0), 128_000)
+			cout, err := NewSpanChannelOut(0, big.NewInt(0), 128_000, Brotli10)
```

Comment:
As long as Fjord is not running on mainnet, let's use `Zlib` as default in such tests.